packages/story-editor/src/components/library/panes/media/local/dropDownMenu.js

```diff
@@ -53,6 +54,7 @@ const MoreButton = styled(Button).attrs({ variant: BUTTON_VARIANTS.ICON })`
   border-radius: 100%;
   width: 28px;
   height: 28px;
+  opacity: ${({ display }) => (display ? '1' : '0')};
```

Comment:
Since this is now tabbable, it needs to be available already before, thus hiding it visually only.